components/VM_Arm/CMakeLists.txt

```diff
@@ -127,6 +127,16 @@ config_option(
     OFF
 )
 
+config_option(
+    VmVirtUart
```

Comment:
I wonder if we should call this `VmVirtConsole` and `VM_VIRT_CONSOLE`, as this seem the primary use case for this.

Reply:
While console is the obvious use case, its just a paravirtualized uart...so it doesn't have to be a console. Also, there's already a `VM_VIRTIO_CONSOLE`, so I think that would be confusing.